src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -189,6 +191,38 @@ public void AddConstant(string name, FormulaValue data)
             }
 
             _variables.Add(name, info); // can't exist
+        }
+
+        /// <summary>
+        /// Adds an user defnied function.
+        /// </summary>
+        /// <param name="script">String representation of the user defined function.</param>
+        /// <param name="parseCulture">CultureInfo to parse the script againts.</param>
+        internal void AddUserDefinedFunction(string script, CultureInfo parseCulture)
```

Comment:
>AddUserDefinedFunction [](http://example.com/codeflow?start=22&length=22)

bind here. 
Probably should take in a ReadOnlySymbolTable here, which we use to bind the body.